src/base/base.pri

```diff
@@ -53,7 +53,8 @@ HEADERS += \
     $$PWD/unicodestrings.h \
     $$PWD/torrentfilter.h \
     $$PWD/scanfoldersmodel.h \
-    $$PWD/searchengine.h
+    $$PWD/searchengine.h \
+    $$PWD/rss/utorrentrssdata.h
```

Comment:
I can, but why? This is a QtCreator project file, what's the point in managing it by hand?